lifecycle/launch/lifecycle_demo.launch.py

```diff
@@ -20,7 +20,7 @@
 def generate_launch_description():
     return LaunchDescription([
         LifecycleNode(package='lifecycle', executable='lifecycle_talker',
-                      name='lc_talker', output='screen'),
+                      name='lc_talker', namespace='', output='screen'),
```

Comment:
Is `namespace=''` the same as `namespace='/'`? If so, the latter seems clearer to me

Reply:
It's the same in this case.
The difference is that `''` can be modified when the launch file is being included in another launch file.

```python3
LaunchDescription([
  ...
  PushRosNamespace('my_ns'),
  IncludeLaunchDescription(path_to_lifecycle_demo_launch_py),
  ...
])
```

See examples of the difference here https://github.com/ros2/launch_ros/blob/master/test_launch_ros/test/test_launch_ros/actions/test_push_ros_namespace.py.